code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -27,12 +27,11 @@
 		var/list/meancritters = list() // list of possible hostile mobs
 		var/list/nicecritters = list() // and possible friendly mobs
 		for(var/T in typesof(/mob/living/simple_animal))

```

Comment:
Can't you avoid the colon operation by typecasting T (var/mob/living/T) here? Or does that not work because T would be a path?